roles/openshift_node/tasks/main.yml

```diff
@@ -37,7 +37,8 @@
 - name: Save firewalld port for OpenShift
   firewalld: port=10250/tcp permanent=true state=enabled
 
+  # Always bounce service to pick up new credentials
 - name: Enable OpenShift
-  service: name=openshift-node enabled=yes state=started
+  service: name=openshift-node enabled=yes state=restarted
```

Comment:
I don't think we want restarted here. If you're trying to bounce it to pick up new creds, we should use the handler when the creds are updated.

From the ansible service module doc  "restarted will always bounce the service."

http://docs.ansible.com/service_module.html